config/config_small_test.go

```diff
@@ -51,6 +51,7 @@ nats_subject = ["spout"]
 nats_subject_monitor = "spout-monitor"
 
 influxdb_address = "localhost"
+influxdb_protocol = "http"
```

Comment:
Please change this to `https` (i.e. not the default value) to confirm that it is being handled correctly